text/0000-generalized-type-ascription.md

```diff
@@ -1067,57 +701,6 @@ to:
 let : LET pat maybe_init_expr ';' ;`
 ```
 
-Finally, the grammar of function definitions is changed:
```

Comment:
Seems like it's at least worth keeping some of this, right?  Because the grammar is no longer `pat:ty`, it's `pat`, with a semantic requirement that it be ascribed at the top level?